src/highlight-support.js

```diff
@@ -69,6 +69,8 @@ const highlightSupport = {
     const elems = editableHost.querySelectorAll(`[data-word-id="${highlightId}"]`)
     for (const elem of elems) {
       content.unwrap(elem)
+      // in Chrome browsers the unwrap method leaves the host node split into 2 (lastChild !== firstChild)
```

Comment:
We should always assume that there are empty text nodes.

Reply:
sure, but the normalize call doesn't hurt in that case. or are you refering to something else?

Reply:
I just womdered whether there's some hidden logic that depends on it. If there's no need I wouldn't apply it as it's a bit inconsistent if we do it only for that event.